code/controllers/subsystem/weather.dm

```diff
@@ -189,6 +191,21 @@ SUBSYSTEM_DEF(weather)
 		if(!is_weather_event)
 			mob.clear_fullscreen("weather")
 
+/datum/controller/subsystem/weather/proc/update_fire_weather()
+	//find all the flamer_fire objects and set their weather_smothering_strength variable so they extinguish quicker
+	var/list/weather_turfs
+	for(var/area/area as anything in weather_areas)
+		for(var/area/subarea as anything in area.related)
+			weather_turfs = get_area_turfs(subarea)
+	for(var/turf/T in weather_turfs)
```

Comment:
why does this bit typecheck?

Reply:
Im assuming you’re talking about the areas; i copied the code from above to remain consistent with the weathered turfs defined there.

if you mean the istype for fire below these lines thats the whole objective of this code, to reassign the variables of all the fire objects in the weathered turfs. I dont want to do it for every object.

Reply:
Oversight, will fix.